spec/string-indices.js

```diff
@@ -0,0 +1,22 @@
+import chai, { expect} from 'chai'
+import chaiChange from 'chai-change'
+import StringIndices from '../src/string-indices'
+
+chai.use(chaiChange)
+
+describe('StringIndices', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(StringIndices).to.be.a('function')
+  })
+
+  it('returns the string in the sentence found at the index', () => {
```

Comment:
a word within a string is a word, technically, and not it's own 'string'. I would rewrite this to say `returns the word at a specified index of a string`